docs/db/changelog/changesets/user-data-store/1.3.x/20240514-refactor-uds-1.3.0.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" author="Roman Strobl">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="uds_document"/>
+            </not>
+        </preConditions>
+        <createTable tableName="uds_document">
+            <column name="id" type="VARCHAR(36)">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="document_type" type="VARCHAR(32)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="data_type" type="VARCHAR(32)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="document_data_id" type="VARCHAR(255)"/>
+            <column name="external_id" type="VARCHAR(255)"/>
+            <column name="document_data" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="attributes" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="encryption_mode" type="VARCHAR(255)" defaultValue="NO_ENCRYPTION">
+                <constraints nullable="false"/>
+            </column>
+            <column name="timestamp_created" type="TIMESTAMP WITHOUT TIME ZONE" defaultValueComputed="NOW()"/>
+            <column name="timestamp_last_updated" type="TIMESTAMP WITHOUT TIME ZONE"/>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="2" author="Roman Strobl">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="uds_document_history"/>
+            </not>
+        </preConditions>
+        <createTable tableName="uds_document_history">
```

Comment:
Create table `uds_document_history` does not define `document_data_id` from `DocumentHistoryEntity`.

Reply:
Apologies, fixed.